pkg/docker/Dockerfile.go1.22

```diff
@@ -8,30 +10,16 @@ LABEL org.opencontainers.image.documentation="https://unit.nginx.org/installatio
 LABEL org.opencontainers.image.vendor="NGINX Docker Maintainers <docker-maint@nginx.com>"
 LABEL org.opencontainers.image.version="1.34.0"
 
-RUN set -ex \
+RUN --mount=type=bind,target=/rust,from=rust,rw \
```

Comment:
Because this is `from=` a stage, it would be mostly OK, but it's technically not officially supported in the DOI build system / dependency calculation, so adding it makes me a little nervous: https://github.com/docker-library/bashbrew/blob/f71d6ef63e1f5abe6d5cb06a62a3e5e42d68566e/cmd/bashbrew/docker.go#L96-L190

(For example, if we accept this, and then the `FROM ... AS rust` gets removed later, we won't necessarily remember the implications of that meaning that this line now implies `from=rust:latest` which will "just work" but not be accounted for correctly in our dependency calculations.)

Reply:
Would using a target that isn't matching an image solve this such as `rust-build`?